Header header
float32 right_x
float32 right_y
float32 left_x
float32 left_y
string right_gesture
string left_gesture
